gulpfile.js

```diff
@@ -1,17 +1,25 @@
 var gulp = require('gulp');
 var del = require('del');
 var rename = require('gulp-rename');
+var merge = require('merge-stream');
 
 var paths = {
-  src: './src/*',
-  html: './src/**/*.html',
-  js: './src/**/*.js',
+  favicon: './favicon.ico',
+  html: './views/*.html',
+  js: './public/js/*.js',
+  css: './public/css/*.css',
+  imgs: './images/([^\s]+(\.(?i)(jpg|png|gif|bmp|ico))$)',
+};
+
+var bower_paths = {
   angular: './bower_components/angular/angular.js',
   firebase: './bower_components/firebase/firebase.js',
   angularfire: './bower_components/angularfire/dist/angularfire.js',
   angularRoute: './bower_components/angular-route/angular-route.js',
-  bootstrap: './bower_components/bootstrap/dist/css/bootstrap.css',
-  bootstrapMap: './bower_components/bootstrap/dist/css/bootstrap.css.map'
+  angularBootstrap: './bower_components/angular-bootstrap/ui-bootstrap.min.js',
+  momentJs: './bower_components/moment/min/moment.min.js',
+  bootstrapCss: './bower_components/bootstrap-css-only/css/bootstrap.min.css',
```

Comment:
You read Js library sourcecode? Just read an Api
On Mar 8, 2015 8:26 PM, "Alex Memering" <notifications@github.com> wrote:

> In gulpfile.js
> <https://github.com/Memeriaj/CarLog/pull/1#discussion_r26011226>:
>
> >    angular: './bower_components/angular/angular.js',
> >    firebase: './bower_components/firebase/firebase.js',
> >    angularfire: './bower_components/angularfire/dist/angularfire.js',
> >    angularRoute: './bower_components/angular-route/angular-route.js',
> > -  bootstrap: './bower_components/bootstrap/dist/css/bootstrap.css',
> > -  bootstrapMap: './bower_components/bootstrap/dist/css/bootstrap.css.map'
> > +  angularBootstrap: './bower_components/angular-bootstrap/ui-bootstrap.min.js',
> > +  momentJs: './bower_components/moment/min/moment.min.js',
> > +  bootstrapCss: './bower_components/bootstrap-css-only/css/bootstrap.min.css',
>
> I am aware of minifying the code however during development i'd rather be
> able to look at the longer version of it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Memeriaj/CarLog/pull/1/files#r26011226>.
>